content/apps/using-openid.md

```diff
@@ -0,0 +1,122 @@
+ask for OpenID scopes during the [Authorization Request](/apps/using-oauth2.html#authorization-request).
+
+```
+
+https://my-pim.cloud.akeneo.com/connect/apps/v1/authorize?
+    response_type=code&
+    client_id=[OAUTH_CLIENT_ID]&
+    scope=openid email read_products write_products&
+    state=[STATE]
+```
+
+::: info
+You can consult the list of [available authentication scopes](/apps/access-scopes.html#available-authentication-scopes).
+:::
+
+## Authenticate after the first Authorization request
+
+If an user is trying to access your App from his Akeneo PIM, and you want to authenticate him, start an
+[Authorization Request](/apps/using-oauth2.html#authorization-request),
+**even if you already are connected to his Akeneo PIM**.
+
+**During this new Authorization Request, you must request all the scopes your App needs**, including the Authorization scopes,
+in addition of the OpenID scopes.  
+
+## Extracting user information from the ID Token
+
+At the end of the OAuth 2.0 process, if your Access Token Request is accepted, you will receive a JSON response with both tokens:
+
+```json
+
+{
+  "access_token": "Y2YyYjM1ZjMyMmZlZmE5Yzg0OTNiYjRjZTJjNjk0ZTUxYTE0NWI5Zm",
+  "token_type": "bearer",
+  "scope": "openid email read_products write_products",
+  "id_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJodHRwOi8vM.XcmmANmSC2RHqWOI"
+}
+```
+
+### Decoding the ID Token
+
+An ID Token is a JWT is composed of 3 parts encoded independently in base64: `header.payload.signature`.
+
+We recommend using one of the libraries listed [here](https://jwt.io/libraries) with support for `RS256` to decode it
+and retrieve the values inside the payload.
+
+::: warning
+It's not recommended to try to decode and verify the ID Token yourself.
+:::
+
+### Payload
+
+Once you've decoded the payload, it will look like this:
+
+```json
+
+{
+  "iss": "https://my-pim.cloud.akeneo.com",
+  "jti": "c76d558d-d10a-4bac-b320-12c22e36b3db",
+  "sub": "c6acd619-8a08-46c2-9a5e-41a175d9149d",
+  "aud": "206f450e-09a1-44ed-a0b3-9dd80f980ace",
+  "iat": 1643029678.467703,
+  "exp": 1643033278.467703,
+  "email": "john.doe@example.com"
+}
+```
+
+| Field       | Description                          |
+|-------------|--------------------------------------|
+| `iss`       | URL of the token issuer              |
+| `jti`       | Unique identifier for the token      |
+| `sub`       | **Unique user id**                   |
+| `aud`       | Id of the OAuth 2.0 client           |
+| `iat`       | Timestamp of token creation          |
+| `exp`       | Timestamp of token expiration        |
+| `email`     | **(optional) Email of the user**     |
+| `firstname` | **(optional) Firstname of the user** |
+| `lastname`  | **(optional) Lastname of the user**  |
+
+Additonal information (`email`, `firstname`, `lastname`, ...) are only present if you requested the corresponding
+authentication scopes and those were approved by the user.
+
```

Comment:
I would add a notice like: "email, firstname and lastname are values that can be edited on the PIM side, they can change over time. The **subject**  claim (aka `sub`) is a unique and immutable technical identifier which is shared among the PIM and the App. You can map this technical identifier coming from the PIM to your user management stack.


:arrow_up: do adapt, but you see what I mean

Reply:
I agree with the first part, I think the first part is overwhelming for consumers of this documentation.
I preferred to be concise and said `sub` is the unique user id.

Reply:
I went with

```
`email`, `firstname` and `lastname` are values that can be edited on Akeneo PIM and are not verified by
Akeneo. The only value that will truly identify an user is his unique user id in the subject claim (`sub`). 
```